dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlanner.cs

```diff
@@ -72,6 +72,8 @@ public async Task<Plan> CreatePlanAsync(string goal)
 
         try
         {
+            // TODO - If a custom function is used to determine which functions to use, the kernel needs to be able to load them on-demand somehow.
```

Comment:
TODO for this PR/design.

Reply:
Adding another function override for loading functions for now. Will see how that looks/feels.